actionmailbox/lib/action_mailbox/engine.rb

```diff
@@ -29,5 +29,11 @@ class Engine < Rails::Engine
         ActionMailbox.ingress = app.config.action_mailbox.ingress
       end
     end
+
+    initializer "action_mailbox.setup" do
+      ActionDispatch::ActionableExceptions.on ActiveRecord::StatementInvalid do |err|
+        raise SetupError if err.to_s.match?(InboundEmail.table_name)
```

Comment:
Should this use `error.message`?

Reply:
Yeah, I'm in the habit of calling `error.to_s` as setting a `@message` is a common mistake during the definition of custom exceptions, but I'll change that to clarify the examples.